src/utils/viewer/viewer.cpp

```diff
@@ -391,8 +418,34 @@ void Viewer::drawEvent() {
   if (sceneID_.size() <= 0)
     return;
 
-  if (physicsManager_ != nullptr)
+  imgui_.newFrame();
+
+  if (physicsManager_ != nullptr) {
+    if (objectIDs_.size() > 0) {
+      ImGui::SetNextWindowPos(ImVec2(10, 30));
+      ImGui::Begin("main", NULL,
+                   ImGuiWindowFlags_NoDecoration |
+                       ImGuiWindowFlags_NoBackground |
+                       ImGuiWindowFlags_AlwaysAutoResize);
+      ImGui::SetWindowFontScale(2.0);
+      if (commandingVelocity) {
+        ImGui::Text("Commanding velocity");
+        setVelocity(objectIDs_.back(), commandVelocity.first,
+                    commandVelocity.second);
+      }
+      // apply anti-gravity force
+      if (commandingAntiGravityForce) {
+        ImGui::Text("Commanding anti-gravity");
+        physicsManager_->applyForce(
+            objectIDs_.back(),
+            Magnum::Vector3{
+                0, float(9.8 * physicsManager_->getMass(objectIDs_.back())), 0},
```

Comment:
Shall we define the gravity coefficient somewhere instead of hard coding it here?